static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -75,13 +83,42 @@ function ConsoleMessage({breadcrumb, isLast}: ConsoleMessageProps) {
       <Icon isLast={isLast} level={breadcrumb.level}>
         {ICONS[breadcrumb.level]}
       </Icon>
-      <Message isLast={isLast} level={breadcrumb.level}>
-        <MessageFormatter breadcrumb={breadcrumb} />{' '}
-      </Message>
+      <MessageWrapper>
+        <Message isLast={isLast} level={breadcrumb.level}>
+          <MessageFormatter breadcrumb={breadcrumb} />
+        </Message>
+        <StyledConsoleTimestamp isLast={isLast} level={breadcrumb.level}>
+          <Hovercard body={`${t('Relative Time')}: ${relativeTimestamp}`}>
+            {getFormattedDate(breadcrumb.timestamp, 'MMM D, YYYY hh:mm:ss A z', {
+              local: false,
+            })}
+          </Hovercard>
+        </StyledConsoleTimestamp>
+      </MessageWrapper>
     </Fragment>
   );
 }
 
+const StyledConsoleTimestamp = styled('div')<{isLast: boolean; level: string}>`
+  padding: ${space(1)};
+  border-left: 1px solid ${p => p.theme.innerBorder};
+  background-color: ${p =>
+    ['warning', 'error'].includes(p.level)
+      ? p.theme.alert[p.level].backgroundLight
+      : 'inherit'};
+  color: ${p =>
+    ['warning', 'error'].includes(p.level)
+      ? p.theme.alert[p.level].iconHoverColor
+      : 'inherit'};
+  ${p => (!p.isLast ? `border-bottom: 1px solid ${p.theme.innerBorder}` : '')};
+`;
```

Comment:
All for consolidating those styles I just wasn't sure given how Common is used in here.